client/src/templates/Challenges/projects/ProjectForm.js

```diff
@@ -34,10 +35,25 @@ export class ProjectForm extends Component {
     this.props.onSubmit();
   }
   render() {
-    const { isSubmitting, isFrontEnd } = this.props;
+    const { isSubmitting, isFrontEnd, externalSite } = this.props;
     const buttonCopy = isSubmitting
       ? 'Submit and go to my next challenge'
       : "I've completed this challenge";
+
+    let solutionPlaceholder = '';
+    if (externalSite) {
+      if (externalSite === 'replit') {
+        solutionPlaceholder = 'https://repl.it/camperbot/solution';
+      } else {
+        solutionPlaceholder =
+          'https://colab.research.google.com/drive/1UCHiRuBLxo013aMuiDXlaP54LsxzrXH3';
+      }
+    } else if (isFrontEnd) {
+      solutionPlaceholder = 'https://codepen.io/camperbot/full/oNvPqqo';
+    } else {
+      solutionPlaceholder = 'https://camperbot.glitch.me';
+    }
```

Comment:
Not a fan of whatever is happing over here, just to generate a Place Holder. I think these challenges should have an object look up map instead like we do for other things.

/cc @freeCodeCamp/dev-team recommendations welcome.

Reply:
How about creating the placeholder in `Show` and passing that in instead of `externalSite`?  `externalSite` doesn't seem to be used for anything else, after all.

Reply:
I implemented this suggestions. See comment below.